src/data/SessionTracker.hh

```diff
@@ -70,7 +70,7 @@ private:
 
     std::vector<GameStats> m_vGameStats;
 
-    long m_nFileWritePosition{};
+    std::streampos m_nFileWritePosition{};
```

Comment:
Made this `std::streampos` to reduce the need of casting, it's implicitly convertible `std::streamoff` (`long long`)